### DISCUSSION TITLE
Custom dictionaries and/or icons

### DISCUSSION THREAD

**Discussion by CestQuoiUnPseudo** [TriliumNext/Trilium#3092] [Q&A]
Hey all, I've been using Trilium 0.54.2 on Pop_OS! 22.04 for about a week at school and it's been working very well. 

Since i'm in Iceland I made myself a "Vocabulary" or glossary page. For a lot of good reasons i'd find the integration of an Icelandic dictionary incredibly useful (checking the orthography, getting rid of the red squiggly lines for example). I found the dictionary files being en-US-9-0.bdic but i haven't found much help online about bdic files.
I was wondering what possibility there was for me to import a dictionary from somewhere else, and also wondering if there would ever be support for a selection of languages? I could help with French translation.

Also since i'm writing already it'd be neat to have more choice for icons. I'm studying natural sciences and having some nature or science themed icons would make my pages even better. If i could import them i'd try my hand at designing some.

Thanks for the amazing software, planning to learn more about it and eventually help somewhat with the project!

**Comment by sigaloid** [comment]
The dictionary spell check is handled by Electron if i recall correctly. Perhaps you can broaden your search for resources related to custom dictionaries in Electron apps.

**Reply by CestQuoiUnPseudo** [reply]
Thank you, that put me in the right direction. Also, just realized there's already a lot of option for languages check and so activated french spell-checking. Still no icelandic available, so i'll keep looking! Weirdly enough Faroese is here so i'll activate it in the mean time even thought they are not similar enough.
